docs/standard/security/cross-platform-cryptography.md

```diff
@@ -92,19 +119,23 @@ RSA key operations are performed by the OS libraries, and the types of key that
 
 .NET does not expose "raw" (unpadded) RSA operations.
 
-The OS libraries are used for encryption and decryption padding. Not all platforms support the same padding options:
+Padding and digest support vary by platform:
 
-| Padding Mode                          | Windows (CNG) | Linux (OpenSSL) | macOS | Windows (CAPI) |
-|---------------------------------------|---------------|-----------------|-------|----------------|
-| PKCS1 Encryption                      | ✔️           | ✔️              | ✔️   | ✔️             |
-| OAEP - SHA-1                          | ✔️           | ✔️              | ✔️   | ✔️             |
-| OAEP - SHA-2 (SHA256, SHA384, SHA512) | ✔️           | ✔️              | ✔️   | ❌             |
-| PKCS1 Signature (MD5, SHA-1)          | ✔️           | ✔️              | ✔️   | ✔️             |
-| PKCS1 Signature (SHA-2)               | ✔️           | ✔️              | ✔️   | ⚠️\*           |
-| PSS                                   | ✔️           | ✔️              | ✔️   | ❌             |
+| Padding Mode                 | Windows (CNG)            | Linux (OpenSSL) | macOS  | iOS | Android | Windows (CAPI) |
+|------------------------------|--------------------------|-----------------|--------|-----|---------|----------------|
+| PKCS1 Encryption             | ✔️                       | ✔️              | ✔️    | ✔️  | ✔️      | ✔️             |
+| OAEP - SHA-1                 | ✔️                       | ✔️              | ✔️    | ✔️  | ✔️      | ✔️             |
+| OAEP - SHA-2                 | ✔️                       | ✔️              | ✔️    | ✔️  | ✔️      | ❌             |
+| OAEP - SHA-3**               | Windows 11 Build 25324+  | OpenSSL 1.1.1+  | ❌     | ❌  | ❌     | ❌             |
+| PKCS1 Signature (MD5, SHA-1) | ✔️                       | ✔️              | ✔️    | ✔️  | ✔️      | ✔️             |
+| PKCS1 Signature (SHA-2)      | ✔️                       | ✔️              | ✔️    | ✔️  | ✔️      | ⚠️\*           |
+| PKCS1 Signature (SHA-3)**    | Windows 11 Build 25324+  | OpenSSL 1.1.1+  | ❌     |  ❌ | ❌     | ❌             |
```

Comment:
Later tables use superscript 1-N instead of multiple asterisks.  Should this table change to that style?